### ISSUE TITLE
[Suggestion] Cauldron Teleportation is Too Good

### ISSUE THREAD

**Issue by DarianLStephens** [Um-Mitternacht/Bewitchment#566]
While it seems that Waystones should be the go-to method of teleporting, with Cauldron Teleportation being more easily-accessible in the early game, there doesn't seem to be a reason to move beyond them and actually make Waystones.
As an example, you make a cauldron, toss in a piece of Dimensional Sand, name it, and it's part of the network permanently, no maintenance or power required. Theoretically infinite destinations, as well.

Now a Waystone, while that doesn't require a setup on the other side (At least, as it stands, for one-way teleportation.), takes altar power and more space for the ritual. While the change that means they are reusable would theoretically make them the number one option, Cauldrons are more versatile, easier to set up, and require no prior infrastructure.

I propose that Dimensional Sand can be depleted from cauldrons, perhaps storing up to 3 pieces as 'charges' that can be replenished at your leisure. This way, there IS some maintenance to them. It will cost you Eyes of Ender to keep it running, but it still works as an early-game teleportation solution, and allows you to quickly set it up, so long as you have a stockpile of Dimensional Sand.
It would also mean that you DO eventually want to move to Waystones, as they don't need to be remade and merely cost MP, which you'll have ample supply of after getting more well-established.

**So, to summarize:**

The problem:

- Cauldrons are a one-time-setup, practically-free teleportation option.
- Waystones take more to make, take power, and more space.
- Cauldrons are therefore the objectively superior teleportation option.

A potential solution:

- Cauldrons should use up the dimensional sand in teleportations, and should be able to store 3 or so pieces at any given time.

Other notes:

- Waystones remain unchanged, as the indefinitely-reusable option that requires power and more setup.

**Comment by DarianLStephens** [comment]
I'm having difficulty finding related commits.
Was this done, or rejected?

**Comment by Sunconure11** [comment]
Rejected on 1.12.2.

**Comment by DarianLStephens** [comment]
Darn.
So what's the plan with the disparity between waystones and cauldrons? I still see little need to make waystones with cauldron teleportation as it is.
The only use I can see for waystones currently is stealth teleportation, so you don't need to set anything up at the destination if you haven't got any plans on also leaving with teleportation.

**Comment by MoriyaShiine** [comment]
I disagree with rejecting this, something needs to be done about it. One idea would be to make the cauldron cost magic depending on the distance, so long range travel becomes impossible if you're too far away.

**Comment by DarianLStephens** [comment]
I could see that.
So long as Waystones remain a flat MP cost, you'd end up using Cauldrons for short-range teleports, and Waystones for long-range, especially if Cauldron MP cost is exponential based on distance.
Specialization without necessarily changing how they function.

**Comment by MoriyaShiine** [comment]
Sun clarified his statement, he means to reject this for 1.12 (as support is ending), however I will add this in the 1.16 version

**Comment by DarianLStephens** [comment]
Ah, that makes much more sense. Looking forward to 1.16!
